src/attrs-table.cc

```diff
@@ -88,11 +92,13 @@ bool AttributesTable::prepare_entry_buffer(const v8::Local<v8::Object>& pt,
     for (u_int32_t i = 0; i < length; ++i) {
         v8::Local<v8::Value> key = Nan::Get(keys, i).ToLocalChecked();
         v8::Local<v8::String> key_str(key->ToString());
-        if (!ignored_attributes_->IsEmpty() && Nan::HasOwnProperty(Nan::New(*ignored_attributes_), key_str).FromJust()) {
+        v8::String::Utf8Value tag(key);
+        std::string tag_str(*tag);
+
+        if (!ignored_attributes_.empty() && _contains(ignored_attributes_, tag_str)) {
```

Comment:
The previous `IsEmpty()` check was just verifying that ignored_attributes_  was actually set to a valid object. That's not possible any more, and iterating over an empty vector is a safe operation, so there's no need for the explicit `empty()` check.